circuit-foundation/src/commonMain/kotlin/com/slack/circuit/foundation/PausableState.kt

```diff
@@ -58,18 +57,40 @@ public fun <T> pausableState(
   isActive: Boolean = LocalRecordLifecycle.current.isActive,
   produceState: @Composable () -> T,
 ): T {
-  var state: T? by remember(key) { mutableStateOf(null) }
-
+  val pauseableState = remember(key) { PauseableState<T>() }.also { it.update(produceState) }
```

Comment:
Does the location of the `.also` block matter here?

Reply:
Nope, doesn't have an consequence on the output